lib/project_types/script/layers/application/create_script.rb

```diff
@@ -36,10 +37,10 @@ def install_dependencies(ctx, language, script_name, project, project_creator)
             ProjectDependencies.install(ctx: ctx, task_runner: task_runner)
           end
 
-          def bootstrap(ctx, project_creator)
+          def bootstrap(ctx, project, project_creator)
             UI::StrictSpinner.spin(ctx.message('script.create.creating')) do |spinner|
               project_creator.bootstrap
-              spinner.update_title(ctx.message('script.create.created'))
+              spinner.update_title(ctx.message('script.create.created', File.join(project.folder, project.source_file)))
```

Comment:
Does `project` not have a helper method on it that can do this already? It already has the information, and if it's `File` oriented already it makes sense. Could be added if it does not and would be useful? Like `project.source_path` or something.

Reply:
Good idea 🙏 I added that